Makefile

```diff
@@ -90,8 +95,18 @@ ifneq (,$(findstring $(ARCH_TAG),x86 x64 arm64))
   CXXFLAGS += -momit-leaf-frame-pointer
 endif
 
+DFlags=
```

Comment:
`DFlags` sounds vague and does not follow naming convention. I'd call it `TEST_FLAGS`

Reply:
This was from PR-812. Addressed and will reflect in next commit